src/main/kotlin/no/nav/familie/ef/sak/integration/JournalpostClient.kt

```diff
@@ -39,6 +42,27 @@ class JournalpostClient(@Qualifier("azure") restOperations: RestOperations,
                 .getDataOrThrow()
     }
 
+    fun hentOvergangsstønadSøknad(journalpostId: String, dokumentInfoId: String): SøknadOvergangsstønad {
+        return getForEntity<Ressurs<SøknadOvergangsstønad>>(jsonDokumentUri(journalpostId, dokumentInfoId)).getDataOrThrow()
+    }
+
+    fun hentBarnetilsynSøknad(journalpostId: String, dokumentInfoId: String): SøknadBarnetilsyn {
+        return getForEntity<Ressurs<SøknadBarnetilsyn>>(jsonDokumentUri(journalpostId, dokumentInfoId)).getDataOrThrow()
+    }
+
+    fun hentSkolepengerSøknad(journalpostId: String, dokumentInfoId: String): SøknadSkolepenger {
+        return getForEntity<Ressurs<SøknadSkolepenger>>(jsonDokumentUri(journalpostId, dokumentInfoId)).getDataOrThrow()
+    }
+
+    private fun jsonDokumentUri(journalpostId: String, dokumentInfoId: String): URI {
+        return UriComponentsBuilder
```

Comment:
Hvis du bruker
UriComponentsBuilder.fromUri(journalpostURI)
                .pathSegment("hentdokument", journalpostId, dokumentInfoId)
                .queryParam("variantFormat", DokumentVariantformat.ORIGINAL)
                .build()
                .toUri()
, så slipper du all string-konkateneringen og transformasjonen av journalpostUri fra URI til String for så å bruke den til å bygger en URI igjen.